server/lib/transforms/insert-ad.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const adMarkup = () => `<div class="ad-container"><amp-ad width="300" height="250" type="doubleclick"></amp-ad></div>`;
+
+module.exports = function insertAd($, options) {
+		const paragraph = $('p').eq(3);
```

Comment:
We have some more complex positioning logic for [light signup](https://github.com/Financial-Times/google-amp/blob/master/server/lib/transforms/light-signup.js#L29-L50) that does similar things to that Next code. Maybe we could split that into its own module so both transforms could use it?